ext/date/php_date.stub.php

```diff
@@ -668,12 +667,14 @@ public static function __set_state(array $array): DateTimeZone {}
 
 class DateInterval
 {
+    public readonly bool $from_string;
```

Comment:
If we're creating it, shouldn't it be `$fromString` now? And ditto with `$date_string` in `DateTimeStringInterval`?

I am also unsure as to the new names...

Reply:
Yeah, I realized a bit too late that the consistent property name would be `$fromString`. On the other hand, this property is only kept for BC concerns (since from now on you'll know it based on the class name), so renaming it defies the main purpose. I'm also fine with removing the property altogether if we think that its BC impact is negligible.

Reply:
Regarding the other names, I'm fine with finding better names/moving them under some namespace :)